ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog for typed-process
 
+## 0.2.7.1
```

Comment:
I think following PVP this should be 0.2.8.

Reply:
Ah, thank you. Fixed!